google-cloud-pubsub/src/main/java/com/google/cloud/pubsub/v1/Publisher.java

```diff
@@ -909,7 +914,7 @@ private static class MessageFlowController {
       this.lock = new ReentrantLock();
 
       this.outstandingMessages = 0L;
-      this.outstandingBytes = 0L;
+      this.outstandingBytes = (long) topicNameSize;
```

Comment:
I'd prefer leaving this at 0, as the flow control mechanism is separate from the batching mechanism. There isn't an API limit that this needs to fit within.